src/templates/project-page.css

```diff
@@ -20,8 +20,9 @@
   flex: 1;
```

Comment:
I pulled all the CSS in this file from PR #63 – if you want to commit changes, feel free!